components/servo/webview.rs

```diff
@@ -399,6 +399,7 @@ impl WebView {
     }
 
     pub fn reload(&self) {
+        self.inner().servo.compositor().notify_reload(self.id());
```

Comment:
It is *very odd* that this only happens on reloads and not on any other type of navigation. Can you justify this a bit in the PR description? Why is this behavior necessary / useful?